.github/workflows/tests.yml

```diff
@@ -50,9 +50,23 @@ jobs:
 
       - name: Precompile assets
         run: bundle exec rake assets:precompile
-      
+
       - name: Run tests
         env:
           RAILS_ENV: test
           DATABASE_URL: postgres://complaint_tracker:postgres@localhost:5432/complaint_tracker_test
-        run: bundle exec rake
+        run: bundle exec rake spec
+
+  standard_ruby:
+    name: Standard Ruby Linter
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+
+      - name: Install dependencies
+        run: bundle install
+      - name: Run linter
```

Comment:
same general newline/consistency question

Reply:
Of course, by my own logic above, there should have been a newline after 70